src/Classes/Utils/Constants.ts

```diff
@@ -131,6 +132,17 @@ export class Constants {
                 value: 20,
             },
         },
+        polygon: {
+            debug: {
+                enable: false,
+                color: "#ffffff",
+            },
+            type: PolygonMaskType.none,
+            move: {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The key 'move' is not sorted alphabetically](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5042590)